cms/djangoapps/course_to_library_import/helpers.py

```diff
@@ -28,7 +28,7 @@
 log = logging.getLogger(__name__)
 
 
-def create_block_in_library(block_to_import, usage_key, library_key, user_id, staged_content_id, override):
+def create_block_in_library(block_to_import, usage_key, library_key, user_id, staged_content_id, task_id, override):
```

Comment:
Let's use `import_id` instead of `task_id`, because a record in `CourseToLibraryImport` is not a task.